features/login.feature

```diff
@@ -0,0 +1,29 @@
+Feature: New customer Registration
+
+
+Background: User is Logged In
+	Given the user navigates to Homepage
+	When the user clicks on the create a account button
+	Then the user should be navigated to customer account page
+
+Scenario: Empty registration Authentication (Negative scenario)
+    When the user clicks on create an account button
+    Then the user should see 5 error message
+    # And "This is a required field" should be displayed
+    When I fill the login form with "<emailid>"
+    Then the user should see Please enter a valid email address Please enter a valid email address Ex: johndoe@domain.com
+    When User enters password with invalid format
+    Then the user should see Minimum of different classes of characters in password is 3. Classes of characters: Lower Case, Upper Case, Digits, Special Characters message
+
+    Examples:
+            | emailid | 
+            | karthikeyan3071997gmail.com | 
+
+Scenario: valid Registration Authentication (Postive scenario)
+    Given The user will navigates to Homepage url
```

Comment:
@karthikeyan7010 What's the purpose of background when you again start from scratch here?